flink-ml-lib/src/main/java/org/apache/flink/ml/feature/vectorassembler/VectorAssembler.java

```diff
@@ -47,7 +47,9 @@
 
 /**
  * A Transformer which combines a given list of input columns into a vector column. Types of input
- * columns must be either vector or numerical value.
+ * columns must be either vector or numerical types. The elements assembled in the same column must
+ * have the same size. If the size of the element is not equal to sizes[columnIdx], it will throw an
+ * IllegalArgumentException.
```

Comment:
It is determined by `HasHandleInvalid` whether to throw exceptions. Let's improve the JavaDoc to describe this behavior.